itest/rust/src/builtin_tests/serde_test.rs

```diff
@@ -9,7 +9,7 @@ use crate::framework::itest;
 use godot::builtin::{array, Array, GString, NodePath, StringName, Vector2i};
 use serde::{Deserialize, Serialize};
 
-fn serde_roundtrip<T>(value: &T, expected_json: &str)
+pub(crate) fn serde_roundtrip<T>(value: &T, expected_json: &str)
```

Comment:
The reason why this (and the surrounding module) was private is so that the checks are self-contained in `serde_test.rs` and we don't need additional `#[cfg]`s elsewhere.

Could you move the serde tests into this file and make the 2 symbols private again?